action.yml

```diff
@@ -1,4 +1,4 @@
-name: "Create GitHub App Token"
+name: "Tim's Create GitHub App Token"
```

Comment:
Why it is `Tim's` ?

Reply:
I think Time wanted to publish the action to the marketplace in order to run the action from the fork, but publishing to the marketplace is not necessary. We definitely need to revert this change before merging

Reply:
Yes, that is exactly why. I'll revert it, post haste.